docker-compose.yaml

```diff
@@ -23,8 +22,6 @@ services:
       PGRST_DB_ANON_ROLE: postgrest_web_anon
       PGRST_JWT_SECRET: $PGRST_JWT_SECRET_LEGACY_SCRIPTS
       PGREST_MAX_ROWS: $PGREST_MAX_ROWS
-    env_file:
-      - ./env
```

Comment:
this is removed because `.env` is magically picked up but `env` had to be specified upon spinning the stack up?

Reply:
never mind, i found the docs